src/WebJobs.Script/Workers/Rpc/RpcWorkerChannelState.cs

```diff
@@ -1,23 +1,31 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
 {
+    [Flags]
     public enum RpcWorkerChannelState
     {
         /// <summary>
         /// The Default state of LanguageWorkerChannel.
         /// </summary>
-        Default,
+        Default = 1,
```

Comment:
>Default = 1, [](start = 8, length = 12)

I think these need to be powers of 2 (I forgot to push that change in my previous iteration)

Reply:
thanks...this was typo. just fixed it.